### ISSUE TITLE
[a11y][Bug] The blue rectangle from talk back is incorrectly displayed on search bar

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#4219]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Launch Fenix.
2. Select Search bar.

### Expected behavior
The blue rectangle from search bar should be correctly displayed and fit the search bar.
### Actual behavior
The blue rectangle from talk back is not correctly displayed when search bar is selected.

* Android device: **`Samsung Galaxy s10+(Android 9)`**
* Fenix version: **`RC v1.1.0-rc.1`**

![Screenshot_20190722-160220_Firefox Preview](https://user-images.githubusercontent.com/42831109/61634650-7175d700-ac9a-11e9-874e-60334b4486f8.jpg)

**Comment by ekager** [comment]
I fixed this for the homescreen in #4496

**Comment by AndiAJ** [comment]
Hi @ekager , this is still reproducible on **debug build 1.0.1934** from 22/8 using a Google Pixel 3a (Android 9)

![rsz_screenshot_20190822-165454](https://user-images.githubusercontent.com/51314259/63520985-44694e00-c4fe-11e9-8fa8-1c5c91ef97d3.png)

**Comment by abodea** [comment]
Hello, I re-tested this issue on the latest Nightly build **12/02** with **Google Pixel 3a XL(Android 10) and Samsung Galaxy S10+(Android 9)** for more details please check the following:

| Device  | Search screen Portrait | Search screen Landscape | Home screen Portrait | Home screen Landscape |
|:-------------:|:-------------:|:-------------:|:-------------:|:-------------:|
| **Samsung S10+(A 9)**  | ❌ | ❌      | ✅ | ✅ |
| **Google Pixel 3a XL(A10)**  | ❌ | ❌      | ✅ | ✅ |

![Screenshot_20191202-123555](https://user-images.githubusercontent.com/42831109/69952919-e4247800-1500-11ea-91ba-56bce7ef4ff0.png)
![Screenshot_20191202-123610](https://user-images.githubusercontent.com/42831109/69952926-e7b7ff00-1500-11ea-8f58-068c52a9390f.png)
![Screenshot_20191202-123604](https://user-images.githubusercontent.com/42831109/69952921-e5ee3b80-1500-11ea-938c-2d393204d760.png)
Based on my comment I will remove the `eng:qa:needed` until further notice.

**Comment by mcarare** [comment]
@abodea This is the expected result, as the rectangle also includes any padding a view has. TY!

**Comment by abodea** [comment]
Thank you @mcarare.
Based on the comments above, I will close this issue.
Verified as fixed on the latest Nightly build **12/02** with **Google Pixel 3a XL(Android 10) and Samsung Galaxy S10+(Android 9)**.
